docs/tiltfile_concepts.md

```diff
@@ -217,22 +218,35 @@ k8s_resource(
 
 #### Kubernetes Object Selectors
 
-Tilt specifies Kubernetes objects via _Kubernetes object selectors_.
-The most specific selector for any object is its _fullname_: a colon-separated
-string specifying `$NAME:$KIND:$NAMESPACE` (e.g.: `redis:deployment:default`).
+You can specify Kubernetes objects via a _Kubernetes object selector_. (NOTE: 
+this is a Tilt-specific syntax. We wish Kubernetes already had a standard for
+specifying objects, but they don't, so we made our own.)
+
+The best-qualified object selector for a given object is a colon-separated
+string of the form `$NAME:$KIND:$NAMESPACE` (e.g.: `redis:deployment:default`).
+We call this the object's _fullname_.
 
 Additionally, you may specify an object using any prefix of its fullname. For above example,
-the other valid selectors for `redis:deployment:default` are:
+the other valid object selectors for `redis:deployment:default` are:
 * "redis"
 * "redis:deployment"
 
-More generically, a Kubernetes object selector is formatted:  `$NAME[:$KIND[:$NAMESPACE]]`.
-
-You may identify an object with the shortest unique selector across all Kubernetes
-objects: e.g. "redis" suffices if there's only one object named "redis", but if
-there's both a Deployment and a Service named "redis", you'd need to specify
-"redis:deployment". You may always use a *more* qualified selector, even if a
-shorter one would suffice.
+More generically, a Kubernetes object selector is formatted:
+```
+$NAME[:$KIND[:$NAMESPACE]]
+```
+(with each successive element being optional).
+
+An object selector is only valid if _uniquely identifies a single object_; that
+is, if it specifies exactly one object across all Kubernetes objects that Tilt
+knows about. For example, the string "redis" suffices if there's only one object
+named "redis", but if there exist both a Deployment and a Service named "redis",
+you'd need to instead use a more qualified object selector like "redis:deployment".
+
+You may always use a _more_ qualified object selector, even if a shorter one would
```

Comment:
I think "would suffice", "would work", and "are valid" are duplicative. I'd pick one and use it consistently.